_pages/rust-hack.md

```diff
@@ -23,12 +23,17 @@ Rust is also a good project to be engaged in for people that don't want or can't
 
 __Impact:__
 
-* 1,000 programmers/developers are introduced to and are well on their way to learning Rust
+* 1,000 programmers are introduced to and are well on their way to learning Rust
 * 10 ongoing Rust Hack and Learn events
+* 500 social media shares
+* Each organizer schedules 3 Rust Hack and Learn events
```

Comment:
on the whiteboard, we wrote 'Met at least ~3 times'. attempted to clarify it a little, but not convinced mine is better

Reply:
I find that better then math notation.